crates/pipeline-manager/src/compiler/rust_compiler.rs

```diff
@@ -636,9 +636,11 @@ async fn prepare_workspace(
         opt-level = 0
         lto = \"off\"
         codegen-units = 256
+        debug = true
```

Comment:
I hope the overhead isn't too high. (But it might be worth it for now anyway.)

Reply:
changed it ti `debug = "line-tables-only"`